mungegithub/mungers/blunderbuss.go

```diff
@@ -90,7 +93,11 @@ func printChance(owners weightMap, total int64) {
 	}
 }
 
-func getPotentialOwners(author string, feats *features.Features, files []*githubapi.CommitFile, leafOnly bool) (weightMap, int64) {
+type ownerVetter interface {
```

Comment:
Yeah, I meant from a testing perspective and from a separation of concerns perspective -- just because we have a mega-super `gihub.Config` object doesn't mean we should pass the whole thing in it's entirety to dependencies that require some small part of it -- the same way that normally in the `Initialize()` specific bits are cherry-picked out to embed into a munger's configuration